docs/dispatcher-install.md

```diff
@@ -72,6 +65,66 @@ Note: ```-auth-version``` is the version of the deployed Openstack Identity Serv
 }
 ```
 
+### Email Notifications
+
+The `$noreply_email` and `$noreply_password` properties are used by SAPS to notify users about job completion. In the current SAPS version, if one uses a gmail account, it is necessary to enable the following gmail configuration:
+
+Sign in to your Gmail account and go to **Manage your Google account**
+![Manage your Google account](img/dispatcher-install-configure-noreply-email-img1.png)
+
+Then, navigate to **Security** settings
+![Security settings](img/dispatcher-install-configure-noreply-email-img2.png)
+
+Look for **Less secure app access** and enable it
+![Less secure app access](img/dispatcher-install-configure-noreply-email-img3.png)
+
+- The `$openstack_object_store_service_key` property in [Dispatcher configuration file](/config/dispatcher.conf) is used to access Object Store. After creating a new key, it must be configured using the command:
+
```

Comment:
lines 82-87 seems to be on the wrong place

Reply:
Done